scripts/release.sh

```diff
@@ -594,6 +598,7 @@ common_help() {
   --tag             <tag>    Explicit tag (overrides the git tag).
   --incremental              Builds components in two stages allowing for faster rebuilds during development.
   --build-bins               Builds all the static binaries.
+  --no-static-linking        Avoid building statically linked binaries.
```

Comment:
Help test makes it sound like the binaries won't be built.
Something like this, feel free to modify:
```suggestion
  --no-static-linking        Don't build the binaries with static linking.
```